services.md

```diff
@@ -34,8 +37,13 @@ manipulation of their state.
 A service communicates with others by exchanging state information about
 entities.
 
+#### What is a **micro** service?
+
+The **micro** in micro service applies to the number of domain concepts a service operates on. All services should endeavour to be small _in scope_, i.e. should only address one (or two) domain concepts and be **very** resistant to adding more.
```

Comment:
> should _endeavour_ to be small _in scope_
covers this, I think. You need to be rigourous in limiting the scope of a micro service, IMO.